instructions/steps.md

```diff
@@ -304,26 +281,24 @@ albeit not always immutable.
 These types already contribute some progress towards correctness.
 After all, you can't put a number into a string.
 
-A subset type in Dafny lets us combine the correctness we defined
-in our `predicate`s with a base `datatype`.
+A subset type in Dafny lets us encapsulate the correctness we defined
+in a `predicate` into a new, more precise type.
 We can then reason about this new type statically.
 If most types represent the basic shape of your data,
-then Dafny's subset type is a painting
+then Dafny's subset types are paintings
 full of light, shadow, and color.
 
 As we will see,
 to return a subset type we will need to prove
 that the `datatype` has been constructed correctly.
-But after that the correctness in baked into the subset type.
+But after that the correctness is baked into the subset type.
 
 Let's create one!
 
 <!-- !test check AwsKmsArn -->
 ```dafny
 
-  type AwsKmsArn = arn: AwsArn
-  | AwsKmsArn?(arn)
-  witness *
+  type AwsKmsArn = arn: AwsArn | AwsKmsArn?(arn) witness *
```

Comment:
sigh, when the formatting gods come and smite you... :)

Reply:
This was a common source of feedback from multiple reviewers and I agree with them. :)

Personally if you REALLY want multiple lines, I'd at least want to indent subsequent lines. But even then there I worry about `| ...` looking too much like a constructor for a datatype.